src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ViewComponentTagHelperOptionalParam.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@addTagHelper "*, AppCode"
+
+<vc:optional-test></vc:optional-test>
+<vc:optional-test show-secret="@true"></vc:optional-test>
+
+<vc:optional-test-with-param secret="mysecret"></vc:optional-test-with-param>
+<vc:optional-test-with-param secret="mysecret" show-secret="@true"></vc:optional-test-with-param>
+
+<vc:optional-with-multiple-types></vc:optional-with-multiple-types>
+<vc:optional-with-multiple-types age="12" favoriteDecimal="10.8" favoriteLetter="a"></vc:optional-with-multiple-types>
```

Comment:
Could you also include 

```
<vc:optional-with-multiple-types favoriteDecimal="10.8" birthDate="DateTime.Utc.Now"></vc:optional-with-multiple-types>
```

So it's a mix of values specified and not-specified?